docs/application-setup.md

```diff
@@ -16,23 +16,27 @@ meta -->
 
 # Quick setup
 
-<iframe width="560" height="315" src="https://www.youtube.com/embed/kEzTZ2U9KMM" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-
-This document shows you how to quickly set up a Luigi web application by installing some of our [examples](https://github.com/SAP/luigi/tree/main/core/examples). 
+<!-- add-attribute:class:warning -->
+> **NOTE:** The quick setup scripts are not intended for production use.  
```

Comment:
Not sure if this might be confusing, as some might think "I can't use my app in production, because I have initially created it with the setup script".
Would rather phrase it like
"The quick setup scripts are a quick and easy way to come to a running core app and micro frontend hosted by the same webserver to play around with but rather don't represent a real-life scenario where the different pieces are distributed"